### PR TITLE
Implement API connect and fetching data

### PR DESCRIPTION
**Pull request by makeychik**
There is some problems with repeating requests when app init

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const proxy = 'https://immense-hamlet-20285.herokuapp.com/';
+const api = 'https://pfa.foreca.com';
+export const url = proxy + api;
+
+export const authData = {
+  user: 'nmakeychik1',
+  password: '64vwAHlJXbJN'
```
Comment (ximet) [inline src/constants.js]: nice, right now I can use your credentials. Thank you)

File: src/App.jsx [hunk 2]
```diff
@@ -1,5 +1,48 @@
...
+import SearchLocation from './components/SearchLocation';
+import { url, authData } from './constants';
+
 function App() {
-  return <div>Hello world!</div>;
+  const [token, setToken] = useState('');
+  const [coords, setCoords] = useState({});
+  const [searchedLocation, setSearchedLocation] = useState({});
+
+  navigator.geolocation.getCurrentPosition(
```
Comment (JuliaShavyrova) [inline src/App.jsx]: do you want to call the function on each render?

File: src/App.jsx [hunk 3]
```diff
@@ -1,5 +1,48 @@
+import React, { useEffect, useState } from 'react'; // нужно ли импортировать в каждый файл?
```
Comment (JuliaShavyrova) [inline src/App.jsx]: please remove question comments
and yes it is ok to import in each file

File: src/App.jsx [hunk 4]
```diff
@@ -1,5 +1,48 @@
...
+      setCoords(pos);
+    },
+    err => {
+      console.warn(`ERROR(${err.code}): ${err.message}`);
+    }
+  );
+
+  const isSearchedLocationEmpty = Object.keys(searchedLocation).length === 0;
+
+  const getSearchedLocation = location => {
```
Comment (JuliaShavyrova) [inline src/App.jsx]: as i can see, your functions getSearchedLocation and getLocation don't really get anything, please rename

File: src/constants.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+const proxy = 'https://immense-hamlet-20285.herokuapp.com/';
```
Comment (JuliaShavyrova) [inline src/constants.js]: please make you own proxy as discussed

File: src/constants.js (hunk 1 above)
Comment (makeychik) [reply src/constants.js]: moved them to .env on the server side

File: src/App.jsx (hunk 2 above)
Comment (makeychik) [reply src/App.jsx]: no, refactored this moment

File: src/App.jsx (hunk 3 above)
Comment (makeychik) [reply src/App.jsx]: done

File: src/App.jsx (hunk 4 above)
Comment (makeychik) [reply src/App.jsx]: I renamed them but not sure about good naming. Need to research best practicies to name callbacks which pass as props

File: src/constants.js (hunk 5 above)
Comment (makeychik) [reply src/constants.js]: done

**Review by JuliaShavyrova — APPROVED** [review]

**Pull request merged**